resources/views/maps/markers/edit.blade.php

```diff
@@ -36,13 +36,14 @@
             <div class="form-group">
                 <div class="submit-group">
                     <input id="submit-mode" type="hidden" value="true"/>
-                    <div class="btn-group">
+                    <div class="pull-right">
+                    <div class="btn-group">                   
```

Comment:
To keep the code clean, please indent with a tab the whole block inside "pull-right".

Reply:
Changes implemented